.goreleaser-dev.yml

```
@@ -18,7 +18,7 @@ release:
   make_latest: false
 
   header: |
-    This is a premature, unstable version built on each commit to the main branch. Use on your own risk.
+    This is a premature, unstable version nightly built from the main branch. Use on your own risk.
```

I'd suggest changing it to
```suggestion
    This is a premature, unstable nightly version built from the main branch. Use on your own risk.
```
or
```suggestion
    This is a premature, unstable version built nightly from the main branch. Use on your own risk.
```